src/udev/udevadm-trigger.c

```diff
@@ -323,7 +292,7 @@ int trigger_main(int argc, char *argv[], void *userdata) {
         }
         r = exec_list(udev_enumerate, action, settle_set);
         if (r < 0)
-                return 1;
+                return r;
```

Comment:
This changes the return convention... That's a break in compat, but hopefully nobody uses this return value. At least it's not documented.